src/modules/roc_pipeline/sender.cpp

```diff
@@ -130,7 +130,7 @@ Sender::Sender(const SenderConfig& config,
     if (config.resampling && config.sample_rate != format->sample_rate) {
         resampler_.reset(new (allocator) audio::ResamplerWriter(
                              *awriter, sample_buffer_pool, allocator, config.resampler,
-                             config.channels),
+                             config.channels, config.poisoning),
```

Comment:
Don't we want to introduce some stupid PoisonReader, PoisonWriter? As it's a
helper mechanism I want the internal components to know about it as less as
possible.

Reply:
I like this idea. Not sure about PoisonWriter, but PoisonReader might work. I'll try both.

Reply:
Could we pass `ReceiverConfig` directly to the `ReceiverSession`. It allows to reduce the number of parameters that `ReceiverSession` constructor takes.

Reply:
I've extracted global receiver output parameters into ReceiverOutputConfig and now we pass it to the ReceiverSession. Passing the whole ReceiverConfig would be a bit confusing because it contains default session config. This default config is currently used for all sessions unconditionally, but in this will change when we implement some kind of session negotiation like SDP.